firmware/motion_board/src/pid_regulator.h

```diff
@@ -0,0 +1,24 @@
+#ifndef PID_REGULATOR_H
+#define PID_REGULATOR_H
+
+typedef struct
+{
+    int32_t reference;
+    int32_t feedback;
+    int32_t error;
+    float integrator;
+    float kp;
+    float ki;
+    float kd;
+    int32_t clamp_min;
+    int32_t clamp_max;
+    float command;
+} PidReg_t;        // pid positional form

```

Comment:
```suggestion
} pid_t;        // pid positional form

```

Ovaj modul nije po underscore, a mozes ga i pojednostaviti, neka bude `pid` umjesto `pid_regulator`